### PR TITLE
add BE udf bitmap_or & bitmap_and

### PR DESCRIPTION
**Pull request by DanyBin**
1.add BE udf bitmap_or & bitmap_and
#2534 #2535

### CODE REVIEW

**Review by kangkaisen — DISMISSED** [review]
+1. LGTM

File: be/src/exprs/bitmap_function.cpp [hunk 1]
```diff
@@ -387,7 +387,48 @@ BigIntVal BitmapFunctions::bitmap_intersect_finalize(FunctionContext* ctx, const
     delete src_bitmap;
     return result;
 }
+StringVal BitmapFunctions::bitmap_or(FunctionContext* ctx, const StringVal& src, const StringVal& dst){
```
Comment (imay) [inline be/src/exprs/bitmap_function.cpp]: return null when input is null.

Comment (imay) [inline be/src/exprs/bitmap_function.cpp]: better to change src and dst to lhs and rhs.

File: be/src/exprs/bitmap_function.cpp [hunk 2]
```diff
@@ -387,7 +387,48 @@ BigIntVal BitmapFunctions::bitmap_intersect_finalize(FunctionContext* ctx, const
...
+        } else {
+            bitmap.merge(RoaringBitmap((char*)dst.ptr));
+        }
+    }
 
+    StringVal result(ctx,bitmap.size());
+    bitmap.serialize((char*)result.ptr);
+    return result;
+}
+StringVal BitmapFunctions::bitmap_and(FunctionContext* ctx, const StringVal& src, const StringVal& dst){
```
Comment (imay) [inline be/src/exprs/bitmap_function.cpp]: rename src and dst to lhs and rhs

File: gensrc/script/doris_builtins_functions.py [hunk 3]
```diff
@@ -610,7 +610,10 @@
         '_ZN5doris15BitmapFunctions12bitmap_countEPN9doris_udf15FunctionContextERKNS1_9StringValE'],
     [['bitmap_empty'], 'BITMAP', [],
         '_ZN5doris15BitmapFunctions12bitmap_emptyEPN9doris_udf15FunctionContextE'],
-
+    [['bitmap_or'], 'BITMAP', ['VARCHAR','VARCHAR'],
```
Comment (imay) [inline gensrc/script/doris_builtins_functions.py]: input should be BITMAP.

**Review by imay — CHANGES_REQUESTED** [review]

**Comment by DanyBin** [comment]
already updated

**Review by imay — APPROVED** [review]
LGTM

**Pull request merged**